src/dmd/traits.d

```diff
@@ -671,6 +672,22 @@ Expression semanticTraits(TraitsExp e, Scope* sc)
 
         return isCopyable(t) ? True() : False();
     }
+    if (e.ident == Id.hasAliasing)
+    {
+        foreach (o; *e.args)
+        {
+            auto t = isType(o);
```

Comment:
The `if` could be rewritten as starting with `if (auto t = isType(o))` which will read simpler.

Reply:
Done.